### DISCUSSION TITLE
Sniff debug functions

### DISCUSSION THREAD

**Discussion by lucraraujo** [squizlabs/PHP_CodeSniffer#3561] [Q&A] [answered]
I wanted to sniff for debug function like `var_dump()`, `dump()` and `dd()`. Is there a way to do that?

**Comment by jrfnl** [comment, marked as answer]
You could use the `Generic,PHP.ForbiddenFunctions` sniff and you can provide your own list via your custom ruleset: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Customisable-Sniff-Properties#genericphpforbiddenfunctions

Alternatively, you could have a look at the [PHP-Var-Dump-Check](https://github.com/php-parallel-lint/PHP-Var-Dump-Check) package, though that is unrelated to PHPCS.
